src/app/components/PopularPlaylist/PopularPlaylist.css

```diff
@@ -0,0 +1,23 @@
+.popular-playlist-header {
+  font-size: 28px;
+  font-weight: normal;
+  color: #fff;
+  margin-bottom: 2%;
+}
+
+.popular-playlist-row {
+  display: flex;
+  gap: 10px;
+  overflow-x: auto;
+  justify-content: space-between;
+}
+
+.popular-playlist-container {
+  margin-bottom: 5%;
+}
+
+@media (max-width: 500px) {
```

Comment:
It's better to remove this max-width with margin, if you will add flex box gap for main container:)